frontend/src/features/public-form/components/FormPaymentPage/stripe/components/PaymentItemDetailsBlock.tsx

```diff
@@ -26,9 +26,9 @@ export const PaymentItemDetailsBlock = ({
       <Text textStyle="body-1" mb="0.75rem">
         {paymentItemName}
       </Text>
-      <Box as="h2" textStyle="h2">{`${centsToDollars(
+      <Box as="h2" textStyle="h2">{`S$${centsToDollars(
```

Comment:
/superultratinynit we might be able to be terser by omitting the template literals here since it's in jsx land
```jsx
<Box as="h2" textStyle="h2">
  S${centsToDollars(paymentAmount ?? 0)}
</Box>
```

Reply:
done!